addons/mail/static/src/core/common/thread.js

```diff
@@ -173,6 +197,18 @@ export class Thread extends Component {
         return this.state.showJumpPresent ? PRESENT_THRESHOLD - 200 : PRESENT_THRESHOLD;
     }
 
+    get preferenceButtonText() {
+        const [, before, inside, after] =
+            _t(
+                "<button>Change your preferences</button> to receive new notifications in your Inbox."
+            ).match(/(.*)<button>(.*)<\/button>(.*)/) ?? [];
+        return {
+            before,
+            inside,
+            after,
+        };
```

Comment:
`{ before, inside, after }` inline _(so the footprint of this function is reduced, with less LOCs)_